understanding/focus-not-obscured-minimum.html

```diff
@@ -231,21 +167,15 @@ <h4>Non-persistent opened information</h4>
                   
                   <section id="user-openable-persistent-disclosures">
                      
-                     <h4>User openable, persistent disclosures</h4>
+                     <h4>利用者が開くことができる、永続的なディスクロージャー</h4>
                      
-                     <p>Some disclosure patterns provide a mechanism for the user to open additional content
-                        that remains open until intentionally closed by the user. Accordions are a simple
-                        example of such a pattern. Chatbots and expandable side navigation are more complex
-                        examples. All of these patterns can be implemented so they are not at risk of failing
-                        this SC. Some possible approaches are:
+                     <p>一部のディスクロージャーパターンでは、利用者が意図的に閉じるまで開いたままになる追加コンテンツを利用者が開くメカニズムを提供する。アコーディオンは、そのようなパターンの簡単な例である。チャットボット及び拡張可能なサイドナビゲーションは、より複雑な例である。これらのパターンはすべて実装できるため、この達成基準に失敗するリスクはない。考えられるアプローチとしては次のようなものがある:
```

Comment:
「複雑」との対比で、「単純」としてみました。
```suggestion
                     <p>一部のディスクロージャーパターンでは、利用者が意図的に閉じるまで開いたままになる追加コンテンツを利用者が開くメカニズムを提供する。アコーディオンは、そのようなパターンの単純な例である。チャットボット及び拡張可能なサイドナビゲーションは、より複雑な例である。これらのパターンはすべて実装できるため、この達成基準に失敗するリスクはない。考えられるアプローチとしては次のようなものがある:
```